internal/services/datafactory/data_factory_dataset_parquet_resource.go

```diff
@@ -76,12 +76,54 @@ func resourceDataFactoryDatasetParquet() *pluginsdk.Resource {
 				},
 			},
 
+			// Parquet Specific Field, one option for 'location'
+			"azure_blob_fs_location": {
+				Type:          pluginsdk.TypeList,
+				MaxItems:      1,
+				Optional:      true,
+				ConflictsWith: []string{"azure_blob_storage_location", "http_server_location"},
```

Comment:
Can we change this to an `ExactlyOneOf`
```suggestion
				ExactlyOneOf: []string{"azure_blob_fs_location", "azure_blob_storage_location", "http_server_location"},
```